examples/diff/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "draftPublishedDiff",
+  "version": "1.0.0",
+  "description": "Diffing UI extension for Contentful's web app.",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "create": "contentful-widget create --space-id $SPACE_ID",
+    "update": "contentful-widget update --space-id $SPACE_ID --force",
+    "dev": "contentful-widget update --space-id $SPACE_ID --force --src 'http://localhost:8001/index.html'"
```

Comment:
The docs start a local server on `3030` and this sets the src to `localhost:8001` which would mean that out of the box the widget would 404 for people. Maybe use the same ports everywhere for consistency.